internal/common/apps.go

```diff
@@ -31,6 +31,8 @@ func RunApps() {
 			defaultAppsAppName := fmt.Sprintf("%s-%s", state.GetCluster().Name, "default-apps")
 
 			if skipDefaultAppsApp {
+				// FIX ME: For private CAPZ tests, comment out the log line below.
+				// It cause a runtime error: invalid memory address or nil pointer dereference. Yet to know why.
```

Comment:
???

Reply:
I yet don't know why it's trigging the panic, but will figure out and let you know.